setup-tdx-host.sh

```diff
@@ -110,7 +110,10 @@ grub_set_kernel
 # just force the installation of modules-extra to make sure we have it
 apt install --yes --allow-downgrades linux-modules-extra-${KERNEL_RELEASE}
 
-grub_cmdline_kvm || true
+# in recent -intel kernel, tdx is enabled by default in kvm_intel
+# so this is not necessary anymore
+#grub_cmdline_kvm || true
```

Comment:
I'm in favor to just remove dead code, but if you feel strongly to keep it, that's okay for me.

Reply:
Yeah, agree, however i kept it to help users to understand this transition without having to read git history, will remove it later